docs/usecases/datastorage_for_institutions.rst

```diff
@@ -157,162 +429,313 @@ about where data are stored, as they can access it just as easily as before.
-and be thus backed-up. Easy-to-reobtain input data can safely be dropped to free
-disk space on the compute cluster again.
+   The key to this lies in flexibly regenerating subdataset's URLs based on their
+   ID and a path to the RIA store. The :command:`datalad get` command is
+   capable of generating RIA URLs to subdatasets on its own, if the higher level
+   dataset containts a ``datalad get`` configuration on ``subdataset-source-candidate-origin``
+   that points to the RIA store the subdataset is published in. Here is how the
+   ``.datalad/config`` configuration looks like for the top-level dataset of the
+   `HCP dataset <https://github.com/datalad-datasets/human-connectome-project-openaccess>`_::
 
-With this remote data store setup, the compute cluster is efficiently used for
-computations instead of data storage. Researchers can not only compute their
-analyses faster and on larger datasets than before, but with DataLad's version
-control capabilities their work also becomes more transparent, open, and reproducible.
+      [datalad "get"]
+          subdataset-source-candidate-origin = "ria+http://store.datalad.org#{id}"
 
+   With this configuration, a :command:`datalad get` can use the URL and insert
+   the dataset ID in question into the ``{id}`` placeholder to clone directly
+   from the RIA store.
 
-.. findoutmore:: Software Requirements
+   The configuration either needs to be done by hand with a :command:`git config`
+   command [#f6]_, or exists automatically in ``.git/config`` if the dataset is
+   cloned from a RIA store.
 
-   - git-annex version 7.20 or newer
-   - DataLad version 0.12.5 (or later), or any DataLad development version more
-     recent than May 2019 (critical feature: https://github.com/datalad/datalad/pull/3402)
-   - The ``cfg_inm7`` run procedure as provided with ``pip install git+https://jugit.fz-juelich.de/inm7/infrastructure/inm7-datalad.git``
-   - Server side: 7z needs to be in the path.
+**Configurations can hide the technical layers**
+
+Setting up a RIA store and appropriate siblings is fairly easy -- it requires
+only the :command:`datalad create-sibling-ria` command.
+However, in the institute this usecase describes, in order to spare users
+knowing about RIA stores, custom configurations are distributed via DataLad's
+run-procedures to simplify workflows further and hide the technical layers of
+the RIA setup:
+
+A `custom procedure <https://jugit.fz-juelich.de/inm7/infrastructure/inm7-datalad/blob/master/inm7_datalad/resources/procedures/cfg_inm7.py>`_
+performs the relevant sibling setup with a fully configured link to the RIA store,
+and, on top of it, also creates an associated repository with a publication
+dependency on the RIA store to an institute's GitLab instance [#f3]_.
+With a procedure like this in place system-wide, an individual researcher only
+needs to call the procedure right at the time of dataset creation, and have a
```

Comment:
"and **has** a"?

Reply:
Thanks! :) I'll add you to the handbook contributors. If you want, feel free to add yourself to the [zenodo.json](https://github.com/datalad-handbook/book/blob/master/.zenodo.json) to be on the author list at the time of the next [release](https://zenodo.org/record/3608612#.Xj1wyKYo9hE).

Hey @all-contributors please add @sarah1989 for review